modules/graderStake/s4GradedBlock.go

```diff
@@ -0,0 +1,64 @@
+package graderStake
+
+import "sort"
+
+// S1GradedBlock is an spr set that has been graded. The set should be read only through it's interface
+// implementation.
+type S4GradedBlock struct {
+	baseGradedBlock
+}
+
+// S4Band is the size of the band employed in the grading algorithm, specified as percentage
+const S4Band = float64(0.01) // 1%
+
+// Version returns the underlying grader's version
+func (g *S4GradedBlock) Version() uint8 {
+	return 8
+}
+
+// WinnerAmount returns the version specific amount of winners.
+func (g *S4GradedBlock) WinnerAmount() int {
+	return 100
+}
+
+// Winners returns the winning SPRs
+func (g *S4GradedBlock) Winners() []*GradingSPR {
+	if len(g.sprs) < 100 {
```

Comment:
switch cnt := len(g.sprs) {
   case cnt < 25:
        return nil
   case cnt > 100:
        return g.sprs[:100]
   default:
        return g.sprs
}

Reply:
I don't think we have to have 100 addresses.  I believe 25 is what we require?